docs/commands/lint.md

```diff
@@ -239,6 +239,28 @@ Use `redocly lint --format=github-actions` to have any encountered problem annot
 ::warning title=operation-operationId,file=museum-with-errors.yaml,line=16,endLine=16,col=5,endColumn=5::Operation object should contain `operationId` field.
 ```
 
+#### Markdown
+
+Use `markdown` format output with the `lint` command to get a markdown-ready output format.
+This is useful if you want to report the outcomes to a document, or use the information in a GitHub Job Summary.
+
```

Comment:
I see there is an explanation for the screenshot, but shouldn't there be some kind of introductory line here like, "The following is an example of the output as it would display in the terminal: or something along those lines

Reply:
Added! (also added the missing screenshot, oops)